docs/project-setup/08-configure-storage.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: /configure-storage
+---
+
+# 8. Configure Storage
+
+A common use case when setting up your project is storing files. Volca includes functionality for storing files in a secure manner with AWS S3 and AWS Cognito.
+
+## Enabling storage
+
+Enabling storage is as simple as enabling it in your `app.config.ts` file.
+
+```ts
+production: {
+    ...
+    storage: {
+      enabled: true,
+    },
+    ...
+  },
+```
+
+### Enable for local development
+
+To enable storage when developing locally you first need to deploy some assets to AWS. We recommend deploying a staging environment and then using it for your local development.
+
+Once the stack is deployed, you can find the name of the bucket in the outputs of the api stack in your terminal. You can then add that value as an environment variable in your local config.
+
+```ts
+local: {
+  ...
+  environmentVariables: {
+    ...
+    AWS_S3_ASSETS_BUCKET: '<your-s3-bucket-name>',
```

Comment:
I think we should stick to either "File" or "Asset" - I prefer file. But let's keep for now.